commit 1f499dd3f067848c344c13e04305e9f428438c9f
Author: Mattia Dongili <malattia@linux.it>
Date:   Fri Jun 5 08:57:06 2009 +1000

    man: explain TapButton default values
    
    Clarify how the new default values are set and what options to set
    to get a different behaviour.
    
    Signed-off-by: Mattia Dongili <malattia@linux.it>
    Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>

Index: xserver-xorg-input-synaptics/man/synaptics.man
===================================================================
--- xserver-xorg-input-synaptics.orig/man/synaptics.man	2009-06-06 12:59:09.725856323 +0900
+++ xserver-xorg-input-synaptics/man/synaptics.man	2009-06-06 13:35:52.040859452 +0900
@@ -813,6 +813,9 @@
 vertical scrolling is enabled, horizontal two-finger scrolling is disabled and
 edge scrolling is disabled. If no multi-finger capabilities are reported,
 edge scrolling is enabled for both horizontal and vertical scrolling.
+Tapping is disabled by default for touchpads with one or more physical buttons.
+To enable it you need to map tap actions to buttons. See the "TapButton1",
+"TapButton2" and "TapButton3" options.
 .LP
 Button mapping for physical buttons is handled in the server.
 If the device is switched to left-handed (an in-server mapping of physical
